.circleci/docker/build.sh

```diff
@@ -112,15 +112,15 @@ case "$image" in
     CONDA_CMAKE=yes
     ;;
   pytorch-linux-focal-py3-clang7-asan)
-    ANACONDA_PYTHON_VERSION=3.7
+    ANACONDA_PYTHON_VERSION=3.9
```

Comment:
Why asan 3.9 but onnx 3.8?

Reply:
Because ONNX needs to run on minimally supported version, while for ASAN we don't care that much